SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security
+
+Target takes the security of our software products and services seriously, which includes all source code repositories managed through [our GitHub organizations](https://opensource.target.com).
+
+If you believe you have found a security vulnerability in any Target-owned repository that meets Target's definition of a security vulnerability, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them to the Target Cyber Security team at _____.
+
+If you prefer to submit without logging in, send an email to _____. If possible, encrypt your message with our PGP key; please download it from _____.
```

Comment:
opensource@target.com is already the reporting mechanism for code of conduct violations. should it continue to be that in this case, or perhaps security@target.com?

@djsudduth do you have thoughts?

Reply:
Agree that the security mailbox would be the appropriate contact mech. Jay L can confirm

Reply:
also reached out to our BISO

Reply:
I also just now became aware of https://security.target.com/vdp which might be a great place to funnel everyone through - pending some internal alignment